README.rst

```diff
@@ -0,0 +1,113 @@
+FIXME - testing correctness of these links
+.. image:: https://travis-ci.org/diffpy/diffpy.pdfmorph.svg?branch=master
+   :target: https://travis-ci.org/diffpy/diffpy.pdfmorph
+
+.. image:: http://codecov.io/github/diffpy/diffpy.pdfmorph/coverage.svg?branch=master
+   :target: http://codecov.io/github/diffpy/diffpy.pdfmorph?branch=master
+
+
+PDFmorph
+========================================================================
+# FIXME
+add description, sample below
+Code package for morphing atomic pair distribution functions.
+
+
+REQUIREMENTS
+------------------------------------------------------------------------
+
+PDFmorph requires Python 3 and several third party libraries 
+that are used by PDFmorph and its components.
+
+* NumPy              - library for scientific computing with Python
+* matplotlib         - Python 2D plotting library
+* SciPy              - library for highly technical Python computing
+* diffpy.utils       - shared helper utilities for wx GUI, 
+https://github.com/diffpy/diffpy.utils
+
+# FIXME - add link for instructions
+If you can't use conda and need to build from the sources, instructions are [here].
+
+INSTALLATION
+------------------------------------------------------------------------
+
+The preferred method is to use Anaconda Python and install from the
+"diffpy" channel of Anaconda packages ::
+
+     conda config --add channels diffpy
+     conda install diffpy.pdfmorph
+
+# FIXME
+PDFmorph can then be started from a terminal...
+[insert more instructions here]
```

Comment:
@chiahaoliu @sbillinge I was thinking that here we could say something like "PDFmorph is a CLI software. After installation is complete, run `pdfmorph <source filename> <target filename>` to see it in action." Would that be appropriate? Suggestions appreciated!

Reply:
great, thanks for this!
That's a yes, but please avoid jargon (do chemists know what CLI means?)

I would have a testing step so something like

"check the installation worked by activating the relevant conda environment and typing
```
pdfmorph --version
```
If it is installed correctly pdfmorph should return its version number"

Then basically your text is fine, though the user may need to know what targe and source filenames refer to.

Imagine you are beginning grad student in chemistry and you want to use this.  When you interact with the documentation you will either have an enjoyable time or a frustrating time.  Your job is to make the document so your target audience (beginning chem grad) is delighted.

Once you have written it, find a suitable person to test it on.  Can they follow the instructions happily without you having to jump in and explain anything?  If they get hung up somewhere, you know you have to add some stuff there.

In order to not maintain two versions of all the software, it might be better to refer to the online documentation in the readme and put the instructions there, though some basic instructions here to get people going are definitely helpful.  Does it make sense?